planner/core/explain.go

```diff
@@ -110,11 +110,17 @@ func (p *PhysicalTableScan) ExplainInfo() string {
 	} else if haveCorCol {
 		fmt.Fprintf(buffer, ", range: decided by %v", p.AccessCondition)
 	} else if len(p.Ranges) > 0 {
-		fmt.Fprint(buffer, ", range:")
-		for i, idxRange := range p.Ranges {
-			fmt.Fprint(buffer, idxRange.String())
-			if i+1 < len(p.Ranges) {
-				fmt.Fprint(buffer, ", ")
+		if p.StoreType == kv.TiFlash {
+			// TiFlash table always use full range scan for each region,
+			// the Ranges here is used to prune cop task
```

Comment:
This is a little misleading. What about `the ranges in p.Ranges is used...`.

Reply:
Fixed